src/main/java/lesson3/task1/Seat.java

```diff
@@ -0,0 +1,44 @@
+package lesson3.task1;
+
+public class Seat {
+    private int seatNumber;
+    private int rowNumber;
+    private boolean available;
+
+    public Seat(int seatNumber, int rowNumber, boolean available) {
+        this.seatNumber = seatNumber;
+        this.rowNumber = rowNumber;
+        this.available = available;
+    }
+
+    public static String reserveSeat(Seat seatToReserve) {
+        if (seatToReserve.available == true) {
+            seatToReserve.available = false;
+            return "You have reserved this seat successfully!";
```

Comment:
It's not a good idea to return String with some (positive / negative) messages. This concept will be very hard to maintain in code

Reply:
It would be much better to return boolean (true/false) to show that state of variable was changed or not. Another good solution would be to return nothing (void method)